packages/gestalt/src/SideNavigation/ItemContent.tsx

```diff
@@ -127,6 +141,7 @@ export default function ItemContent({
   return (
     <Box
       alignItems="center"
+      aria-disabled={disabled}
```

Comment:
@mkmatheson Can you check why it still responds to the click? 
![Uploading Brave Browser - SideNavigation - Gestalt 2024-12-05 at 12.16.00 AM.gif…]()